src/controls/selectors/SearchBar.tsx

```diff
@@ -13,7 +14,7 @@ const SearchBar: React.FC = () => {
   const getSearchSuggestions = useSearchSuggestions();
 
   return (
-    <Selector selectorLabel="🔎">
+    <Selector selectorLabel={<Search size="12px" />}>
```

Comment:
Generally we should avoid absolute px measurements for text-like elements. Since 1em=16px then this should be 0.75em.

I'm playing around with other solutions to this though since I think the icon should be bigger (1em) but that's causing unfavorable rendering errors.

Reply:
```suggestion
    <Selector selectorLabel={<Search size="1em" />}>
```

I found that when I set the size to "1em" or "16px", the search icon looks a bit too large and out of proportion with the input field's text, as you can see in the screenshot below.

<img width="547" height="57" alt="image" src="https://github.com/user-attachments/assets/bbc35b40-ee06-4bc5-a3b5-91957995ebe8" />

To solve this while still using relative units, I've adjusted the size to "0.75em". I think this size fits the design much better. Let me know what you think!

<img width="532" height="49" alt="image" src="https://github.com/user-attachments/assets/2fe2d096-230d-4bf7-a5de-7f7340644d03" />